boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -72,6 +72,10 @@ qemu-app-gdb : $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
 	$(OPENTITAN_TREE)/bazel-bin/sw/host/opentitantool/opentitantool.runfiles/lowrisc_opentitan/sw/host/opentitantool/opentitantool --interface=cw310 bootstrap $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
 
+update_registers:
+	$(OPENTITAN_TREE)/bazelisk.sh build //sw/device/tock:tock_lowrisc_registers
+	tar -C $(TOCK_ROOT_DIRECTORY) -xf $(OPENTITAN_TREE)/bazel-bin/sw/device/tock/tock_lowrisc_registers.tar
+
```

Comment:
`update-registers` should probably be in its own section of the Makefile, maybe at the bottom. It doesn't really belong with all the "run in X environment" targets, like "flash" and "flash-app".

Reply:
I didn't even realize that snuck into the Makefile.  I'll eliminate it.